packager/certcache/certcache.go

```diff
@@ -58,8 +73,13 @@ type CertHandler interface {
 
 type CertCache struct {
 	// TODO(twifkak): Support multiple cert chains (for different domains, for different roots).
-	certName          string
-	certs             []*x509.Certificate
+	certName string
+	certsMu  sync.RWMutex
+	certs    []*x509.Certificate
+	// If certFetcher is not set, that means cert auto-renewal is not available.
+	certFetcher       *certfetcher.CertFetcher
+	renewedCertsMu    sync.RWMutex
```

Comment:
1. The certName should change on a cert renewal (the sha256 computed by util.CertName is different), I think? So I think we need a CertCache.renewedCertName field also.
2. I think we need to put any reads/writes of certName and renewedCertName under their respective RWMutex.